lsst_dashboard/gui.py

```diff
@@ -596,7 +604,7 @@ def _assemble_query_expression(self, ignore_query_expr=False):
 
         return query_expr
 
-    def get_dataset_by_filter(self, filter_type, metrics):
+    def get_dataset_by_filter(self, filter_type, metrics, visits=None):
```

Comment:
This method is now given a list of selected visits (I think we only allow one). Should add a condition like:

```python
if visits and self.coadd_toggle.value == 'Visit View':
    return load_visit_data(filter_type, metrics, visits[0])
```